zebra/zebra_dplane.c

```diff
@@ -3988,6 +3991,8 @@ static bool dplane_work_pending(void)
 	struct zebra_dplane_ctx *ctx;
 	struct zebra_dplane_provider *prov;
 
+	return true;
```

Comment:
This seems like some debug stuff left in place?

Reply:
yes, sorry - should have flagged the PR with draft or don't-merge. I was trying to poke at the shutdown failure in the CI in another direction - that there's some pattern where the zebra main and dplane pthreads don't behave as I think they should, and they don't make normal shutdown progress. I want to use this to try a couple of things.